src/components/20-molecules/policy-features/story.js

```diff
@@ -0,0 +1,91 @@
+/* global document */
+import { storiesOf } from '@storybook/html';
+import { radios, select, text, withKnobs } from '@storybook/addon-knobs';
+import './policy-features-item/index';
+import './index';
+import Readme from './README.md';
+
+const story = storiesOf('Molecules/Policy features', module);
+story.addDecorator(withKnobs);
```

Comment:
Please add noBorder cause this component is full with component so that demo is same as real case.

https://github.com/axa-ch/patterns-library/blob/1f5565ed8e264ff1e81819e2c417cdeb50534868/src/components/30-organisms/footer/story.js#L10